.github/workflows/playwright.yml

```diff
@@ -0,0 +1,116 @@
+name: Playwright Visual Regression Test
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    # Prepare testing environment
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
```

Comment:
Add comment which step takes long time. If possible move it to the step which takes long time.